CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 All notable changes to this project will be documented in this file.
 
+[Full Changelog](https://github.com/nextcloud/nextcloud-vue/compare/v7.0.0-beta.0...v6.0.0-beta.7)
+
+### :rocket: Enhancements
+
+- Fix for border-radius being too large for 3-line entries like in Mail [\#3136](https://github.com/nextcloud/nextcloud-vue/pull/3136) ([jancborchardt](https://github.com/jancborchardt))
+- Add related resources panel component [\#3081](https://github.com/nextcloud/nextcloud-vue/pull/3081) ([Pytal](https://github.com/Pytal))
+- Add a native datetime picker [\#3063](https://github.com/nextcloud/nextcloud-vue/pull/3063) ([JuliaKirschenheuter](https://github.com/JuliaKirschenheuter))
+
+### :bug: Fixed bugs
+
+- Fix spacing in app navigation and make the setting pill style [\#3158](https://github.com/nextcloud/nextcloud-vue/pull/3158) ([juliushaertl](https://github.com/juliushaertl))
```

Comment:
This was not backported, and is not in stabel6. Although it's also not correct here afaik.